Automattic/jetpack
# Sync: Delete jetpack-sync-dashboard.php

Remove the jetpack sync dashboard in favor of Syncing within Calypso.

cc @enejb @gravityrail 

Closes #4071

#### Testing instructions:
- checkout this branch.
- go to  `/wp-admin/admin.php?page=jetpack-sync?page=jetpack-sync` on your site and check that the page doesn't exist.
- navigate wp-admin including Jetpack dashboard and make sure nothing is broken
